test/index.ts

```diff
@@ -2,28 +2,190 @@ import axios from 'axios';
 import { expect } from 'chai';
 import * as dotenv from 'dotenv';
 import { startServer } from '../server';
+import { AppDataSource, clearDB } from '../src/data-source';
+import { User } from '../src/entity/User';
 
 const endpoint = 'http://localhost:3000/';
 
-describe('Array', function () {
-  before('starting server', async function () {
+function defaultInput() {
+  return {
+    name: 'Bob Semple',
+    email: 'bobsemple@gmail.com',
+    password: 'pass1234',
+    birthDate: '01-01-1990',
+  };
+}
+
+describe('createUser mutation tests', async () => {
+  before(async () => {
     dotenv.config({ path: './test.env' });
     await startServer();
   });
 
-  it('should get a hello back from the server', async function () {
-    const sayHelloQuery = {
-      operationName: 'helloQuery',
-      query: `query helloQuery { hello }`,
-      variables: {},
+  after(async () => {
+    await AppDataSource.destroy();
+  });
```

Comment:
A config do dotenv e setup do server no início, assim como a desconexão do banco ao final podem ser feitas uma vez só pra todos os testes. Recomendo vc fazer isso no before/after do index em vez de fazer no teste de uam mutation só.